doc/conf.py

```diff
@@ -280,11 +280,19 @@ def make_carousel_thumbs(app, exception):
 issues_user_uri = 'https://github.com/{user}'
 
 
+def maps_any_missing_reference_to_code_block(app, env, node, contnode):
+    if node['reftype'] == 'any':
+        contnode['classes'] = []
```

Comment:
Can you explain (and perhaps add a comment) how this results in code-like styling? Thanks.